crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -992,7 +997,10 @@ impl<T: Component> ChangeTrackers<T> {
 #[doc(hidden)]
 pub struct ChangeTrackersFetch<'w, T> {
     // T::Storage = TableStorage
-    table_ticks: Option<ThinSlicePtr<'w, UnsafeCell<ComponentTicks>>>,
+    table_ticks: Option<(
+        ThinSlicePtr<'w, UnsafeCell<Tick>>, // Added
```

Comment:
Can we put this in a struct without taking a perf hit? Those comments +  a tuple type make me nervous about dumb mistakes later.

Reply:
Done.